src/ansys/fluent/core/solver/flobject.py

```diff
@@ -126,6 +132,16 @@ def flproxy(self):
             return self._parent.flproxy
         return self._flproxy
 
+    @property
+    def remote_file_handler(self):
+        """Remote file handler.
+
+        Supports file upload and download.
+        """
+        if self._remote_file_handler is None:
+            return self._parent.remote_file_handler
+        return self._remote_file_handler
```

Comment:
If we use `return self._remote_file_handler or self._parent.remote_file_handler` then we get recursion error.